Documentation/ceph-pool-crd.md

```diff
@@ -106,6 +106,6 @@ The `failureDomain` must be also be taken into account when determining the numb
 * `host`: All chunks will be placed on unique hosts
 * `osd`: All chunks will be placed on unique OSDs
 
-If you do not have a sufficient number of hosts or OSDs for unique placement the pool can be created, although a PUT to the pool will hang.
+If you do not have a sufficient number of hosts or OSDs for unique placement the pool can be created, a PUT to the pool will hang.
```

Comment:
I don't think everybody can tell what a `a PUT to the pool` means, so I'd clarify that. Something `putting objects in the pool`.